src/components/note/noteDetail/NoteDetail.tsx

```
@@ -27,10 +27,18 @@ const goalColor: Record<string, string> = {
 interface Props {
   noteId: number;
   isModal?: boolean;
+  embedVisible?: boolean;
+  setEmbedVisible?: (visible: boolean) => void;
```

ESLint

[error] 'visible' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)